### PR TITLE
5 create categorycontroller

### PR DESCRIPTION
**Pull request by Anton20HD**
Added CRUD in category controller

### CODE REVIEW

File: src/main/java/com/example/springbootlab/controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import com.example.springbootlab.entity.Category;
+import com.example.springbootlab.repository.CategoryRepository;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("category")
```
Comment (jonathanjonathan001) [inline src/main/java/com/example/springbootlab/controller/CategoryController.java:12]: It's supposed to be a / before category, and may I suggest you change it to plural, categories.

**Review by jonathanjonathan001 — CHANGES_REQUESTED** [review]
Good job! 
I tested it with Insomnia, and I get an error (infinite recursion, because of our bi-directional many-to-one) when i run GET, this can be remedied by adding 

@JsonBackReference in Category.java

so that it looks like this:
    @OneToMany(mappedBy="categoryId", fetch = FetchType.EAGER)
    @JsonBackReference
    private List<Movie> movies;

Can you add this?

(also see my comment on line 12 in CategoryController.java)

**Review by jonathanjonathan001 — APPROVED** [review]
Great!

**Comment by Anton20HD** [comment]
Closes #5

**Pull request merged**